FEM-2666 Upgrade ExoPlayer 2.10.6

GouravSna opened the pull request:
- Added onIsPlayingChanged from ExoPlayer

GouravSna committed:
no message

GouravSna committed:
Merge branch 'dev' into FEM-2666

GouravSna committed:
- Code cleanup

giladna commented on playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java:
```diff
@@ -481,6 +482,13 @@ public void onLoadingChanged(boolean isLoading) {
+    @Override
+    public void onIsPlayingChanged(boolean isPlaying) {
+        if (isPlaying) {
+            sendDistinctEvent(PlayerEvent.Type.PLAYING);
```
@GouravSna 
is there any change in the behaviour if PLAYING event due to this update?

GouravSna replied:
From event prospective PLAYING is being sent as usual.

OrenMe commented on playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java:
```diff
@@ -981,7 +985,8 @@ public float getVolume() {
         log.v("isPlaying");
         if (assertPlayerIsNotNull("isPlaying()")) {
-            return player.getPlayWhenReady() && (currentState == PlayerState.READY || currentState == PlayerState.BUFFERING);
+            return player.getPlayWhenReady() && (currentState == PlayerState.READY || currentState == PlayerState.BUFFERING)
```
let's switch this to use only the new exoplayer API and remove our logic

GouravSna committed:
- Upgraded to ExoPlayer 2.10.6
- Added onPlaybackSuppressionReasonChanged method from ExoPlayer (Only log is being printed)
- isPlaying() is now dependent on both ExoPlayer and our logic
